### PR TITLE
Fix #8326, add back --quiet option.

### LINKED ISSUE #8326: @babel/cli can't be quietened
# v7 Regression

**Potential Commit/PR that introduced the regression**

https://github.com/babel/babel/pull/7439

**Describe the regression**

`@babel/cli` can no longer be quietened.

The `--quiet` flag was removed and a "Successfully compiled..." message is logged to the console without an option to suppress it:

https://github.com/babel/babel/blob/62ee1adc8543975f1ff0d41dd2ad6875404713a7/packages/babel-cli/src/babel/dir.js#L126-L130

**Environment**

`@babel/cli v7.0.0-beta.53`

### PR DESCRIPTION
**Pull request by chris-peng-1244**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #8326 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

I added the `--quiet` to the babel-cli which overrides the `--verbose` option.

### CODE REVIEW

File: packages/babel-cli/src/babel/dir.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default async function({
       outputFileSync(dest, res.code);
       util.chmod(src, dest);
 
-      if (cliOptions.verbose) {
+      if (cliOptions.verbose && !cliOptions.quiet) {
```
Comment (nicolo-ribaudo) [inline packages/babel-cli/src/babel/dir.js]: I'd prefer to throw if both are enabled

Comment (chris-peng-1244) [reply packages/babel-cli/src/babel/dir.js]: @nicolo-ribaudo Do you still want me to throw an error here?

**Comment by nicolo-ribaudo** [comment]
Thanks! We will merge this PR when releasing Babel 7.7.0

**Comment by chris-peng-1244** [comment]
Glad I managed to contribute!

**Comment by woodcockjosh** [comment]
this is so annoying

**Comment by danielhaim1** [comment]
Don't forget to add `--quiet` back to docs

**Pull request merged**